apps/web/vibes/soul/primitives/banner/index.tsx

```diff
@@ -17,71 +17,51 @@ import { ForwardedRef, forwardRef, ReactNode, useCallback, useEffect, useState }
  *   --banner-close-icon-hover: hdl(var(--foreground));
  *   --banner-close-background: transparent;
  *   --banner-close-background-hover: hsl(var(--background)/40%);
+ *   --banner-font-family: var(--font-family-body);
  * }
  * ```
  */
-export const Banner = forwardRef(
-  (
-    {
-      id,
-      children,
-      hideDismiss = false,
-      className,
-      onDismiss,
-    }: {
-      id: string;
-      children: ReactNode;
-      hideDismiss?: boolean;
-      className?: string;
-      onDismiss?: () => void;
-    },
-    ref: ForwardedRef<HTMLDivElement>,
-  ) => {
-    const [banner, setBanner] = useState({ dismissed: false, initialized: false });
+export function Banner({
+  children,
+  hideDismiss = false,
+  className,
+  onDismiss,
+  dismissed = false,
+  initialized = true,
+  ref,
+}: {
+  children: ReactNode;
+  hideDismiss?: boolean;
+  onDismiss?: () => void;
+  dismissed?: boolean;
+  initialized?: boolean;
+  className?: string;
+  ref?: Ref<HTMLDivElement>;
+}) {
+  if (!initialized) return null;
 
-    useEffect(() => {
-      const hidden = localStorage.getItem(`${id}-hidden-banner`) === 'true';
-
-      setBanner({ dismissed: hidden, initialized: true });
-    }, [id]);
-
-    const hideBanner = useCallback(() => {
-      setBanner((prev) => ({ ...prev, dismissed: true }));
-      localStorage.setItem(`${id}-hidden-banner`, 'true');
-      onDismiss?.();
-    }, [id, onDismiss]);
```

Comment:
Are we just dropping this logic? Catalyst currently depends on this so we need to make sure we have an alternative approach to persisting hiding the banner. If could be handled outside of this component, but we should probably put that in an example